packages/search-ui/src/SearchDriver.ts

```diff
@@ -455,6 +459,17 @@ class SearchDriver {
           }
         },
         (error) => {
+          if (error.message === INVALID_CREDENTIALS) {
```

Comment:
Assuming that an expired token puts the search state into an error state, one thing we should consider is the experience this has to our customers. In my opinion, I would say an invalid token should continue to show previous search results & show login modal overlaid. Treating it like an explicit error might show the login + error state.

Reply:
I agree. That's the way it works now.
![Screen Cast 2022-03-03 at 9 51 03 AM](https://user-images.githubusercontent.com/11838280/156622712-34c73351-0114-4309-9bc5-426555d7f865.gif)

Maybe we could not make it an Error in the code as well.